src/modules/AuthModule/components/ResetPass/ResetPass.tsx

```diff
@@ -1,9 +1,278 @@
-import React from 'react'
+import { Visibility, VisibilityOff } from "@mui/icons-material";
+import {
+  Alert,
+  IconButton,
+  InputAdornment,
+  InputLabel,
+  OutlinedInput,
+} from "@mui/material";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import CssBaseline from "@mui/material/CssBaseline";
+import Grid from "@mui/material/Grid";
+import Paper from "@mui/material/Paper";
+import Typography from "@mui/material/Typography";
+import { ThemeProvider, createTheme } from "@mui/material/styles";
+import * as React from "react";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { Link, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import { axiosInstance } from "../../../../axiosConfig/axiosInstance";
+import { FormDataResetPass } from "../../../../interfaces/Auth";
+import {
+  emailValidation,
+  passwordValidation,
+} from "./../../../../validations/validations";
+
+// TODO remove, this demo shouldn't need to reset the theme.
+const defaultTheme = createTheme();
 
 const ResetPass = () => {
+  const navigate = useNavigate();
+  const [showPassword, setShowPassword] = React.useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = React.useState(false);
+
+  const handleClickShowPassword = () => setShowPassword((show) => !show);
+
+  const handleClickShowConfirmassword = () =>
+    setShowConfirmPassword((show) => !show);
+
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<FormDataResetPass>();
+
+  const validatePasswordMatch = (value: unknown) => {
+    const password = watch("password");
+    return value === password || "Confirm Password doesn't match Password";
+  };
+
+  function handleMouseDownPassword(event: React.MouseEvent<HTMLButtonElement>) {
+    event.preventDefault();
+  }
+  function handleMouseDownConfirmPassword(
+    event: React.MouseEvent<HTMLButtonElement>
+  ) {
+    event.preventDefault();
+  }
+
+  const backgroundStyle = {
+    backgroundSize: "cover",
+    backgroundPosition: "center",
+    backgroundRepeat: "no-repeat",
+    display: { xs: "none", sm: "flex" },
+    flexDirection: "column",
+    justifyContent: "flex-end",
+    alignItems: "center",
+  };
+
+  const onSubmit: SubmitHandler<FormDataResetPass> = async (data) => {
+    // setLoadingBtn(true);
+    try {
+      const response = await axiosInstance.post(
+        "/admin/users/reset-password",
+        data
+      );
+      toast.success(response.data.message || "Password changed successfully");
+      navigate("/login");
+    } catch (error: any) {
+      toast.error(error.response.data.message || "error");
+    }
+    console.log(data);
+  };
+
   return (
-    <div>ResetPass</div>
-  )
-}
+    <>
+      <ThemeProvider theme={defaultTheme}>
```

Comment:
The theme should be moved to the app. The cssBaseline as well, cssBaseline as similar to normailzation.css files that we used to add default css styles like box-sizing border box and margin and padding 0. So it shouldn't be repeated in every component.